.gitignore

```diff
@@ -4,7 +4,6 @@
 /.glide
 /.dev
 .DS_Store
-/vendor
```

Comment:
since you don't commit any thing in vendor we don't need to include it, do we?

Reply:
Correct, vendor is useless now, we'll not have any vendor files anymore. That's why I excluded it from gitignore

Reply:
Okay this just means we all need to delete `vendor` going forward 👍

Reply:
We also need to `export GO111MODULE=on` if we are working on kubeaudit in $GOPATH